src/lib.rs

```diff
@@ -4,6 +4,102 @@
 //! standard library. These macros provide extremely useful outputs and can be used for
 //! writing better tests.
 
+/// Asserts that the expression is true
+///
+/// # Examples
+/// ```
+/// # #[macro_use] use all_asserts;
+/// let a = false;
+/// #[cfg(should_panic)]
+/// assert_true!(a);
+///
+/// ```
+/// # #[macro_use] use all_asserts;
+/// let a = true;
+/// assert_true!(a);
+/// ```
+///
+#[macro_export]
+macro_rules! assert_true {
```

Comment:
This is the same as [`std::assert`](https://doc.rust-lang.org/std/macro.assert.html)

Reply:
It is likely that there is a much simpler implementation of assert_true!() and assert_false!(), deriving more directly from assert!(). I will look into this.
Whether assert_true! and assert_false! are useful is discussed in another comment.

Reply:
`assert` is a `#[rustc_bultin_macros]` item; it is built-in to the compiler and is injected into the crates before lowering to high-level IR (HIR)

Reply:
You are correct that assert_false and assert true are redundant with assert.
So is every other assert... function. They all derive from assert and are all directly replaceable with the bare assert. But it is better to build tests at a higher abstraction from the foundation of bare asserts.

Reply:
I'm sorry, I didn't mean to put in a "flame war"; I was just suggesting that the `assert!` macro is used in most cases for _asserting_ boolean exps

Reply:
No "flame war" intended. The interleaving of messages was coincidental. I was continuing the thought responding to your original response.
This is your library and I happily accede to your preferences.